ExampleMod/ExampleMod.cs

```diff
@@ -26,6 +28,55 @@ public class ExampleMod : Mod
 			RandomBuffKeybind = null;
 		}
 
+		public override object Call(params object[] args) {
+			// The following code allows other mods to "call" Example Mod data.
+			// This allows mod developers to access Example Mod's data without having to set it a reference.
+			// Mod calls are not exposed by default, so it will be up to you to publish appropriate calls for your mod, and what values they return.
+
+			// Make sure the call doesn't include anything that could potentially cause exceptions.
+			if (args is null) {
+				throw new Exception("Arguments cannot be null!");
+			}
+			if (args.Length <= 0) {
+				throw new Exception("Arguments cannot be empty!");
+			}
+
+			// Make sure that the argument is a string using pattern matching.
+			// Since we only need one parameter, we'll take only the first item in the array.
+			if (args[0] is string content) {
```

Comment:
```suggestion
			// This check makes sure that the argument is a string using pattern matching.
			// Since we only need one parameter, we'll take only the first item in the array.
			if (args[0] is string content) {
```